Advice - added Linda-d advice page

Linda-D-design opened the pull request.

redrambles commented:
Hey @Linda-D-design,

This is a good start! There are a few issues that we need to resolve, though - before we can merge this in.

First - there is a `advice/your-name-advice.html` file that was added, which we don't need - so please remove that.

Second - your added some CSS (which is great!) but unfortunately you renamed and modified the existing style.css file that many students are using. (oops!) No worries! Let's return the css/style.css file back to its original state (have a look [here](https://github.com/skillcrush/105-practice/blob/master/css/style.css) for reference - if you want to copy/paste the contents)

Once these things are done, please add (`git add -A`) and commit as per usual (`git commit -m "Your message here"`) and then push up your advice branch once more, which will automatically update this pull request. Then we should be all set to merge in your code.

Let us know if you have any questions!

Cheers,
Ann :)

Linda-D-design commented:
I’ve tried to do this, this is the response I get, I had changed the advice branch 

Linda@MacBook-Pro 105-practice % git fetch upstream
Enter passphrase for key '/Users/Linda/.ssh/id_rsa': 
Linda@MacBook-Pro 105-practice % git checkout advice
Already on 'advice'
Linda@MacBook-Pro 105-practice % open .
Linda@MacBook-Pro 105-practice % git add -A
Linda@MacBook-Pro 105-practice % git commit -m "corrected css/html files"
On branch advice
nothing to commit, working tree clean
Linda@MacBook-Pro 105-practice % git push origin advice
Enter passphrase for key '/Users/Linda/.ssh/id_rsa': 
To github.com:Linda-D-design/105-practice.git
 ! [rejected]        advice -> advice (fetch first)
error: failed to push some refs to 'git@github.com:Linda-D-design/105-practice.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.



> On Jun 4, 2020, at 1:17 PM, Ann Cascarano <notifications@github.com> wrote:
> 
> 
> Hey @Linda-D-design <https://github.com/Linda-D-design>,
> 
> This is a good start! There are a few issues that we need to resolve, though - before we can merge this in.
> 
> First - there is a advice/your-name-advice.html file that was added, which we don't need - so please remove that.
> 
> Second - your added some CSS (which is great!) but unfortunately you renamed and modified the existing style.css file that many students are using. (oops!) No worries! Let's return the css/style.css file back to its original state (have a look here <https://github.com/skillcrush/105-practice/blob/master/css/style.css> for reference - if you want to copy/paste the contents)
> 
> Once these things are done, please add (git add -A) and commit as per usual (git commit -m "Your message here") and then push up your advice branch once more, which will automatically update this pull request. Then we should be all set to merge in your code.
> 
> Let us know if you have any questions!
> 
> Cheers,
> Ann :)
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/skillcrush/105-practice/pull/1249#issuecomment-639021817>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/APSEZ5R4ZQ37LCDVSR6PD2DRU7QLLANCNFSM4NNMT2VA>.
>
